pmm-app/src/pmm-qan/panel/QueryAnalytics.tsx

```diff
@@ -50,7 +68,13 @@ const QueryAnalyticsPanel: FC = () => {
   }, [querySelected]);
 
   return (
-    <div className="query-analytics-grid" id="antd" ref={queryAnalyticsWrapper}>
+    // Force remount of the whole QAN subtree when theme mode changes.
+    <div
+      key={grafanaTheme.type}
```

Comment:
If we didn't force the remount, would the colors not work correctly (I'm asking since there are more to theme handling than previously when I've looked at it, maybe it's not needed anymore?)